code/modules/atmospherics/machinery/components/binary_devices/circulator.dm

```diff
@@ -90,8 +90,48 @@
 	if(generator)
 		disconnectFromGenerator()
 	to_chat(user, "<span class='notice'>You [anchored?"secure":"unsecure"] [src].</span>")
+
+	var/obj/machinery/atmospherics/node1 = nodes[1]

```

Comment:
Use the nullify node helper instead of writing this out by hand. Unless there's some reason for this.

Reply:
nullify node helper broke machine.
This code i take from thermomachine.